fn/values.yaml

```diff
@@ -58,7 +70,13 @@ fn_lb_runner:
 fn_api:
   image: fnproject/fnserver
   tag: latest
-  resources: {}
+  resources:
```

Comment:
I confirmed that these work, though you may want to add resource requests/limits for ui for the sake of completeness.

Reply:
thanks, will add.

Reply:
done